markdown/blog/coc-report.md

```diff
@@ -0,0 +1,66 @@
+
+## What Happened?
+A conversation started in our `#mentorship` channel, where certain community members raised concerns about the GSoC selection process for mentees. What initially appeared to be a heated discussion soon evolved into a coordinated effort involving multiple fake and impersonated Slack accounts.
+
+These accounts were used to:
+
+- Harass selected mentees and accused them of unfair selection.
+- Manipulate the discussion
+- Create confusion and break trust within the community.
+
+## Investigation & Findings
+The Code of Conduct Committee, composed of Maya, Ace, Thulie, and Łukasz, convened multiple times to review messages, examine account metadata, and speak with both mentors and mentees affected by the incident.
+
+**Through our investigation, we confirmed that**:
+
+- Several of the accounts involved were not authentic and had been created for the sole purpose of disruption.
+
+- Two individuals, already known members of the community, created separate identities to carry out harassment and impersonation. These accounts were used to post inflammatory messages and manipulate conversations anonymously. After investigation and outreach, both individuals confirmed ownership of the accounts.
+
+- A mentor unintentionally contributed to the confusion by communicating about GSoC selection independently from the organizing team, which fueled speculation and misinformation.
+
+## Action Taken
+
+1. Account Bans
+The fake accounts were permanently removed from the Slack workspace. One individual, after being identified, issued an apology and remains in the community. The other did not respond to outreach and has been banned from the platform. We have also recommended their removal from the GSoC program.
+
+2. Mentor Accountability
+The CoC team has formally warned the mentor involved in the off-platform communications. While they intended to support and onboard contributors, their actions unintentionally contributed to confusion within the community. The CoC emphasized that future concerns regarding mentorship and selection processes should be directed to the official mentorship lead, Ace, to ensure transparency and consistency. Given the circumstances and the mentor’s good intentions, the committee has decided to handle this matter with leniency while providing clear guidance moving forward.
+
+3. Support for Affected Individuals
+All mentees and mentors affected were contacted directly by the CoC team to ensure they felt safe, supported, and heard.
+
+## What We Learned
+This incident highlighted the importance of better CoC visibility and process clarity. It also underscored how easily trust can be shaken when communication is fragmented or unofficial.
```

Comment:
```suggestion
This incident highlighted the importance of better CoC visibility and process clarity. It also emphasized how easily trust can be shaken when communication is fragmented or unofficial.
```

Reply:
I don't think that's the tone Maya wanted to use. I would like the original contents not changed, especially the ones approved by the CoC, since its their report @bandantonio